aws_eden_cli/cmdline.py

```diff
@@ -116,6 +123,91 @@ def command_config(args: dict):
         else:
             logger.info(f"Found {errors} errors")
 
+    elif args['push']:
+        dynamodb = boto3.client('dynamodb')
+
+        table_name = "eden"
+
+        try:
+            response = dynamodb.describe_table(TableName=table_name)
+            table_status = response['Table']['TableStatus']
+        except botocore.exceptions.NoCredentialsError:
+            logger.error("AWS credentials not found!")
+            return
+        except Exception as e:
+            if hasattr(e, 'response') and 'Error' in e.response:
+                if e.response['Error']['Code'] != 'ResourceNotFoundException':
+                    logger.error(f"Unknown exception raised on DescribeTable API: {e.response}")
+                    return
+            else:
+                logger.error(f"Unknown exception raised on DescribeTable API: {e}")
+                return
+
+            response = dynamodb.create_table(
```

Comment:
I'm curious what would happen here if the client has insufficient permission to create a new DynamoDB table.

Reply:
User well see a stack trace from boto3. We could remove the stack trace and only output 'insufficient permissions' to logger.error and exit (as awscli does), but as you can see from other parts of the code, catching boto3 exceptions is too difficult (hint: boto3 errorfactory).

Reply:
Thanks for your explanation. I just care about users can notice a missing permission and handle it on their own by looking into a stack trace in that situation, but disregard my comment. (Not worth to do that hospitably.)

Reply:
Will fix.